test/typecheck/atomic-no-shared-memory.txt

```diff
@@ -0,0 +1,83 @@
+;;; TOOL: wat2wasm
+;;; ARGS: --enable-threads
+
+(module
+  (memory 1)
```

Comment:
Is this because we are changing the spec?  I'd expect to see this change along with a testsuite update.  Don't we have threads spec tests that check this?

Reply:
Hm, looks like the tests weren't updated.

Reply:
https://github.com/WebAssembly/threads/pull/150